src/Debugger/Engine/Impl/AD7Property.cs

```diff
@@ -313,7 +314,7 @@ public AD7Property(AD7Property parent, DebugEvaluationResult result, bool isSynt
             if (fields.HasFlag(enum_DEBUGPROP_INFO_FLAGS.DEBUGPROP_INFO_VALUE)) {
                 if (valueResult != null) {
                     // TODO: handle radix
-                    dpi.bstrValue = valueResult.GetRepresentation().Deparse;
+                    dpi.bstrValue = valueResult.GetRepresentation().Deparse.ToUnicodeQuotes();
```

Comment:
This one is not directly related to the fix, but it's another place where this call was missing where it should have been present.